lib/rules/td-headers-attr.json

```diff
@@ -16,8 +16,8 @@
   ],
   "actIds": ["a25f45"],
   "metadata": {
-    "description": "Ensure that each cell in a table that uses the headers attribute refers only to other cells in that table",
-    "help": "Table cells that use the headers attribute must only refer to cells in the same table"
+    "description": "Ensure that each cell in a table using the headers attribute refers only to header cells within that table",
```

Comment:
I updated this upon @dbjorge request. This caused to changes in rule-descriptions.md should i commit it too?

Reply:
Yes, please include it in the same PR. We do have [a workflow](https://github.com/dequelabs/axe-core/blob/develop/.github/workflows/update-generated-files.yaml) that will auto-update it as a backup for cases where it's missed, but it's easier to read the file change history if it's updated in the same PR with the rest of the context for the change.